lib/osf-components/addon/components/inline-list/component.ts

```diff
@@ -23,4 +23,9 @@ export default class InlineList extends Component {
         }
         return this.items.slice(1, -1);
     }
+
+    @computed('items', 'truncate')
```

Comment:
Should actually depend on `items.[]` since the comparison below is to `items.length`.